providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/process/custom_ops.py

```diff
@@ -31,23 +31,35 @@ def fractional(data: dict, *args: JsonLogicArg) -> typing.Optional[str]:
         logger.error("No hashKey value resolved")
         return None
 
-    hash_ratio = abs(mmh3.hash(bucket_by)) / (2**31 - 1)
-    bucket = int(hash_ratio * 100)
+    hash_ratio = abs(mmh3.hash(bucket_by)) / (2 ** 31 - 1)
+    bucket = hash_ratio * 100
 
+    totalWeight = 0
```

Comment:
Python variables should always be snake_case not camelCase. 

```suggestion
    total_weight = 0
```

Reply:
my inner java dev resurfaced here ;)